docs/SRIOV.md

```diff
@@ -0,0 +1,260 @@
+* Compatible Network Interface Card (NIC)
+
+> When SR-IOV capable NICs are used in conjunction with Open vSwitch bridges, you have the option of using an existing provider label, such as **external**, or using a dedicated provider. When sharing a provider network between SR-IOV and non-SR-IOV ports, communication between the ports on the same network is permitted. Using a dedicated provider will require you to call out a second bridge mapping, such as `sriov:br-sriov`, to allow DHCP ports connected to a vSwitch to communicate with the SR-IOV ports.
+
+### Kernel IOMMU Support
+
+Using **dmesg**, you can verify if IOMMU is enabled with the following command:
+
+```
+# dmesg | grep IOMMU
+```
+
+If you do not see the message ```DMAR: IOMMU enabled```, then proceed with the following steps:
+
+First, enable IOMMU support in the kernel by modifying the GRUB configuration at **/etc/default/grub**:
+
+```
+GRUB_CMDLINE_LINUX="... intel_iommu=on" #Intel-based Systems
+
+GRUB_CMDLINE_LINUX="... amd_iommu=on" #AMD-based Systems
+```
+
+Next, update GRUB:
+
+```
+update-grub
+```
+
+> Once the kernel configuration has been modified, you must reboot for the changes to take effect.
+
+### IOMMU Passthrough Support
+
+To enable IOMMU passthrough support in the kernel, please complete the following steps:
+
+First, enable passthrough support in the kernel by modifying the GRUB configuration at **/etc/default/grub**:
+
+```
+GRUB_CMDLINE_LINUX="... iommu=pt"
+```
+
+Then, update GRUB:
+
+```
+update-grub
+```
+
+> Once the kernel configuration has been modified, you must reboot for the changes to take effect.
+
+## Deploying PMO with SR-IOV support using Express
+
+Using the Platform9 Express tool, operators can deploy PMO with support for SR-IOV. The Express tool will perform many of the tasks outlined in the previous sections, including enabling IOMMU and passthrough support in the kernel, as well as implementing a unit file for persisting VFs across reboots.
+
+> Given the complexity involved in supporting Mellanox NICs, the Express tool will initially only support Intel NICs using the ixgbe driver. This includes the Intel X520, X540, and X550 families.
+
+The necessary configuration details can be implemented globally using **group_vars**, or on an individual host basis using **host_vars**. Each method is described below.
+
+### Host Variables
+
+Compute node-specific configurations can be implemented using what is known as host_vars. Configurations that may vary between hosts include:
+
+* Network interface name
+* Quantity of network interfaces used for SRIOV
+* Provider network mappings
+* Number of VFs per interface
+ 
+Using **host_vars**, the following are some variables that can be modified:
+
+* physical_device_mappings (required)
+* sriov_numvfs (required)
+* neutron_ovs_bridge_mappings (optional)
+
+In this example, two hosts have different NICs installed that report different names to the operating system.
+
+```
+root@compute01:~# ip link show
```

Comment:
Rename to hv01 to align with readme / sample inventory file

Reply:
Fixed.